.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,21 @@
+name: PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  docker-push:
+    runs-on: ubuntu-latest
```

Comment:
Should these be pinned to version 18.04 in line with other repos?

Reply:
Will pin it now, thanks!